### PR TITLE
Update Start-SPOTenantRename.md

### PR DESCRIPTION
**Pull request by WayneEwington**
Updated to state that we are now in Public Preview (versus Private Preview).

### CODE REVIEW

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

**Review by dariomws — APPROVED** [review]

File: sharepoint/sharepoint-ps/sharepoint-online/Start-SPOTenantRename.md [hunk 1]
```diff
@@ -15,8 +15,7 @@ ms.reviewer:
 ## SYNOPSIS
 
 > [!IMPORTANT]
-> - This feature is available only for customers who have been accepted into the Private Preview program. [Request to join the Private Preview](https://aka.ms/SPOTenantRenameNomination). Note that submitting a nomination does not guarantee acceptance into the program. We review the nominations on a case-by-case basis and will contact you if you're accepted.
-> - This feature is not available for organizations that have set up multi-geo.
+> This feature is is in preview and currently available to organizations that have no more than 1000 total SharePoint sites and OneDrive accounts combined.
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Start-SPOTenantRename.md:18]:
```suggestion
> This feature is in preview and currently available to organizations that have no more than 1000 total SharePoint sites and OneDrive accounts combined.
```

**Review by tseward — APPROVED** [review]
Approved with @get-itips changes in place.

File: sharepoint/sharepoint-ps/sharepoint-online/Start-SPOTenantRename.md (hunk 1 above)
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Start-SPOTenantRename.md:18]:
```suggestion
> This feature is in preview and currently available to organizations that have no more than 1,000 total SharePoint sites and OneDrive accounts combined.
```

**Review by JohanFreelancer9 — COMMENTED** [review]
@WayneEwington, kindly commit my suggested changes, which include @get-itips's correction. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**